.github/workflows/build-test.yml

```diff
@@ -35,8 +35,9 @@ jobs:
         name: ${{ matrix.cloud }} JDBC ${{ matrix.category }} on ${{ matrix.image }}
         runs-on: ubuntu-latest
         strategy:
+            fail-fast: false
             matrix:
-                image: [ 'jdbc-centos6-default' ]
+                image: [ 'jdbc-centos6-default', 'jdbc-centos7-openjdk8', 'jdbc-centos7-openjdk11', 'jdbc-centos7-openjdk17' ]
```

Comment:
Do we still need image on centos6? Which version of java is inside?

Reply:
Yes, it would be better to remove old build. We only have to confirm it with Ilesh.

Reply:
Lets see whether we support centos6. If not, we can think of removing it.

Reply:
Ok, so as long as my images cover Java 8 on Centos 7 - maybe we can remove Centos 6? It is no longer maintained and in NodeJS driver we dropped Centos 6 builds.

Reply:
ok